easybuild/tools/options.py

```diff
@@ -907,6 +911,10 @@ def postprocess(self):
         if self.options.deprecated:
             build_log.CURRENT_VERSION = LooseVersion(self.options.deprecated)
 
+        # Convert this to an empty list if not set to simplify usage
+        if self.options.silence_deprecation_warnings is None:
+            self.options.silence_deprecation_warnings = []
+
```

Comment:
is this needed, since you're already changing the default in L499-501? I guess it doesn't hurt either way

Reply:
I forgot to revert that change of the default. I decided against it as it looks ugly. Or maybe it did work because the default of `None` was still used in the "reverse-map"(?) in easybuild/tools/config.py for `silence_deprecation_warnings`

Changed the default back to `None` to be safe here.

Reply:
@jfgrimm After doing the quick fix I (was) remembered why it was this way: I didn't want the `--help` to show `" ... (... default: )`, i.e. an empty default value which is shown for an empty list but not for `None`. Hence I cheated a bit to make it look pretty. But I just added a proper way of handling this.

Reply:
ah, that makes sense